### PR TITLE
feat(wildfly): change jboss bundle documentation to wildfly

### PR DESCRIPTION
**Pull request by redboul**
Closes [BS-15555](https://bonitasoft.atlassian.net/browse/BS-15555)

### CODE REVIEW

File: md/wildfly-bundle.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
-# JBoss bundle
+# Wildfly bundle
 
-You will find here steps needed to install and configure a JBoss bundle.
+You will find here steps needed to install and configure a Wildfly bundle.
 
-The JBoss bundle is a zip archive that contains the Red Hat JBoss JEE application server packaged with Bonita BPM Portal and [Bonita configuration folder](BonitaBPM_platform_setup.md#platform_setup_tool).  
-The JBoss bundle is a regular zip archive based on the JBoss zip distribution.
+The Wildfly bundle is a zip archive that contains the Red Hat Wildfly JEE application server packaged with Bonita BPM Portal and [Bonita configuration folder](BonitaBPM_platform_setup.md#platform_setup_tool).
```
Comment (educhastenier) [inline md/wildfly-bundle.md]: instead of saying "and [Bonita configuration folder]", I would rather be more generic: "ready to use with Boita BPM", because there is not only the setup tool shipped with the bundle.

File: md/wildfly-bundle.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+The Wildfly bundle is based on a standard Wildfly installation with the following additions:
 
 * `bin/standalone.conf`: script to configure JVM system properties.
 * `bonita-start.bat`: script to start the bundle on Windows.
 * `bonita-start.sh`: script to start the bundle on Linux.
-* `standalone/configuration/standalone.xml`: JBoss context configuration for Bonita BPM Portal. It defines data sources used by Bonita BPM Engine.
+* `standalone/configuration/standalone.xml`: Wildfly context configuration for Bonita BPM Portal. It defines data sources used by Bonita BPM Engine.
 * `lib/bonita`: extra libraries needed by Bonita BPM. The following libraries are included: Bitronix JTA Transaction Manager, h2, SLF4J (required by Bitronix).
 * `request_key_utils`: script to generate license request keys (Subscription editions only).
 * `standalone/deployments/h2.sar`: application that runs the h2 server. (h2 is the default embedded database, not for production)
```
Comment (educhastenier) [inline md/wildfly-bundle.md]: not anymore with H2 in autoserver mode

**Comment by educhastenier** [comment]
Also, you must remove "3. Remove the `standalone/deployments/h2.sar` file"

File: md/wildfly-bundle.md [hunk 3]
```diff
@@ -143,22 +143,22 @@ As an example, if hexadecimal parameter is equal to 6. The corresponding binary
 
 #### Shutdown
 
-JBoss can be shut down by running the following script:
+Wildfly can be shut down by running the following script:
 
-* Windows `<JBOSS_HOME>\bin\jboss-cli.bat --connect --command=:shutdown`
-* Linux `<JBOSS_HOME>/bin/jboss-cli.sh --connect --command=:shutdown`
+* Windows `<WILDFLY_HOME>\bin\jboss-cli.bat --connect --command=:shutdown`
```
Comment (educhastenier) [inline md/wildfly-bundle.md]: Does it mean we should remove bonita-stop.sh / .bat for the Wildfly bundle?

File: md/database-configuration-for-business-data.md [hunk 4]
```diff
@@ -66,14 +66,14 @@ The data sources for business data objects are independent of the Bonita BPM Eng
 ::: info
 If you have multiple tenants that use business data objects, you need to create a `BusinessDataDS` and a `NotManagedBizDataDS` for each tenant.
 :::
-The following sections show how to configuire the data sources for [JBoss](#ds_jboss) and [Tomcat](#ds_tomcat). 
+The following sections show how to configuire the data sources for [Wildfly](#ds_wildfly) and [Tomcat](#ds_tomcat).
```
Comment (educhastenier) [inline md/database-configuration-for-business-data.md]: typo "configure"

File: md/database-configuration-for-business-data.md [hunk 5]
```diff
@@ -66,14 +66,14 @@ The data sources for business data objects are independent of the Bonita BPM Eng
 ::: info
 If you have multiple tenants that use business data objects, you need to create a `BusinessDataDS` and a `NotManagedBizDataDS` for each tenant.
 :::
-The following sections show how to configuire the data sources for [JBoss](#ds_jboss) and [Tomcat](#ds_tomcat). 
+The following sections show how to configuire the data sources for [Wildfly](#ds_wildfly) and [Tomcat](#ds_tomcat). 
 There is also an [example of how to configure data sources for Weblogic](red-hat-oracle-jvm-weblogic-oracle.md).
 
-<a id="db_jboss"/>
+<a id="db_wildfly"/>
```
Comment (educhastenier) [inline md/database-configuration-for-business-data.md]: Did you mean to name the anchor **ds_wildfly** ?

File: md/database-configuration.md [hunk 6]
```diff
@@ -244,10 +244,9 @@ First make sure that you have [configured Business Data](database-configuration-
...
 
-* For JBoss
-   * Delete the `$JBOSS_HOME/standalone/deployments/h2.sar` folder.
-   * Remove the configuration for h2 from `<JBOSS_HOME>/standalone/configuration/standalone.xml`.
-   * Make sure that `sysprop.bonita.db.vendor` property in `<JBOSS_HOME>/standalone/configuration/standalone.xml` is not set to h2\.
+* For Wildfly
+   * Remove the configuration for h2 from `<WILDFLY_HOME>/standalone/configuration/standalone.xml`.
+   * Make sure that `sysprop.bonita.db.vendor` property in `<WILDFLY_HOME>/standalone/configuration/standalone.xml` is not set to h2\.
 * For Tomcat
    * Remove h2 jar files (`bonita-jboss-h2-mbean-1.0.0.jar`, `bonita-tomcat-h2-listener-1.0.1.jar`, `h2-1.3.170.jar`). Files are located in: `<TOMCAT_HOME>/lib` or in `<TOMCAT_HOME>/lib/bonita`.
```
Comment (educhastenier) [inline md/database-configuration.md]: Shouldn't `bonita-jboss-h2-mbean-1.0.0.jar`, `bonita-tomcat-h2-listener-1.0.1.jar` already be removed from default installatio, with AUTOSERVER mode?

File: md/database-configuration.md [hunk 7]
```diff
@@ -244,10 +244,9 @@ First make sure that you have [configured Business Data](database-configuration-
...
    * Remove the h2 listener, so that h2 is not started automatically: comment out the h2 listener in the `/conf/server.xml` file.
```
Comment (educhastenier) [inline md/database-configuration.md]: Should already be done.

File: md/performance-tuning.md [hunk 8]
```diff
@@ -138,9 +138,8 @@ If you are running Bonita BPM Engine inside a container, the maximum number of c
 * **Apache Tomcat** `maxThreads` set in _`Tomcat_folder`_`/conf/server.xml`.    
      Default value 200\. 
      See the [Tomcat documentation](http://tomcat.apache.org/tomcat-7.0-doc/) for information about the `maxThreads` parameter.
-* **Red Hat JBoss** `maxThreads` set in _`JBoss_folder`_`/server/default/deploy/jbossweb.sar/server.xml`.   
-     Default value 200\. 
-     See the [JBoss documentation](http://docs.jboss.org/jbossas/guides/webguide/r2/en/html/ch02.html) for information about the `maxThreads` parameter.
+* **Red Hat Wildfly** : add the attributes `io-threads="10"` and `task-max-threads="200"` in the _default_ *worker* element in the io subdomain in _`<WILDFLY_HOME>/standalone/configuration/standalone.xml`.
```
Comment (educhastenier) [inline md/performance-tuning.md]: there is an extra _

File: md/wildfly-bundle.md (hunk 3 above)
Comment (redboul) [inline md/wildfly-bundle.md]: there is already no `bonita-stop` in wildfly

File: md/database-configuration-for-business-data.md (hunk 5 above)
Comment (redboul) [inline md/database-configuration-for-business-data.md]: yes

File: md/database-configuration.md (hunk 6 above)
Comment (redboul) [inline md/database-configuration.md]: yes but it is not part of this PR

File: md/database-configuration.md (hunk 7 above)
Comment (redboul) [inline md/database-configuration.md]: yes but it is not part of this PR for Tomcat

**Comment by tnguyen1** [comment]
retest this please

File: md/wildfly-bundle.md (hunk 3 above)
Comment (educhastenier) [inline md/wildfly-bundle.md]: you're right, I went out of my mind.

File: md/database-configuration-for-business-data.md [hunk 9]
```diff
@@ -66,14 +66,14 @@ The data sources for business data objects are independent of the Bonita BPM Eng
 ::: info
 If you have multiple tenants that use business data objects, you need to create a `BusinessDataDS` and a `NotManagedBizDataDS` for each tenant.
 :::
-The following sections show how to configuire the data sources for [JBoss](#ds_jboss) and [Tomcat](#ds_tomcat). 
+The following sections show how to configure the data sources for [Wildfly](#ds_wildfly) and [Tomcat](#ds_tomcat).
```
Comment (educhastenier) [inline md/database-configuration-for-business-data.md]: Again, #ds_wildfly and #ds_tomcat do not exist
=> db_wildfly, db_tomcat

File: md/database-configuration-for-business-data.md (hunk 5 above)
Comment (educhastenier) [inline md/database-configuration-for-business-data.md]: So, just do it?

**Comment by educhastenier** [comment]
Ok, good.
